types/express-serve-static-core/index.d.ts

```diff
@@ -423,8 +424,10 @@ export interface Request extends http.IncomingMessage, Express.Request {
      */
     xhr: boolean;
 
-    //body: { username: string; password: string; remember: boolean; title: string; };
-    body: any;
+    /**
+     * The parsed request body.
+     */
+    body: unknown;
```

Comment:
This minor change isn't worth making all `express` users upgrade to ts3.0 which was released only two weeks ago.

Reply:
@andy-ms What about generic `Request` with defaults to any that requires TS 2.3? Maybe it's time to move on with the definition and use features from newer TS versions?

Reply:
I was under the impression that different TypeScript versions requirements where published to different tags on npm. That way, users of an older version of TypeScript can stay on that version. [Here you can see the tags, e.g. `ts3.0`](https://www.npmjs.com/package/@types/express)

Have I misunderstood something?

I also think it would be very nice to get this in because I'm seeing a lot of code in the wild which just blindly uses `req.body` with the json-parsing middleware without doing any kind of validation 😱 

Changing this to unknown would really help to nudge people in the right direction towards validating their inputs.

Reply:
The tags do exist, but it seems that most users don't know about them and our documentation doesn't mention them (Microsoft/TypeScript-Handbook#807). We might be updating the whole repository to ts2.4 in the near future though. (@rbuckton)
As for `any`, there is a tslint rule [no-unsafe-any](https://palantir.github.io/tslint/rules/no-unsafe-any/) that tries to look for unsafe property accesses on `any`, though it isn't perfect. I don't know if we're planning to eventually change every `any` on DefinitelyTyped to `unknown`.